### PR TITLE
Add at-rule @keyframes compat data

### PR DESCRIPTION
**Pull request by ddbeck**
This PR adds one at-rule, [@keyframes](https://developer.mozilla.org/en-US/docs/Web/CSS/@keyframes). Since it's the first of it's kind, I figured I should open a PR for this one alone.

### CODE REVIEW

File: css/at-rules/keyframes.json [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+            "ie_mobile": {
+              "version_added": null
+            },
+            "opera": [
+              {
+                "version_added": "12.10"
+              },
+              {
+                "prefix": "-o-",
+                "version_added": "12"
```
Comment (teoli2003) [inline css/at-rules/keyframes.json]: "version_removed": "15"

File: css/at-rules/keyframes.json [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "ie_mobile": {
+              "version_added": null
+            },
+            "opera": [
+              {
+                "version_added": "12.10"
```
Comment (teoli2003) [inline css/at-rules/keyframes.json]: "version_removed": "15"

(Opera lost support for unprefixed CSS Animations when moving to Chromium/Blink

File: css/at-rules/keyframes.json [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+              },
+              {
+                "prefix": "-o-",
+                "version_added": "12"
+              }
+            ],
+            "opera_android": {
```
Comment (teoli2003) [inline css/at-rules/keyframes.json]: Same as Opera.

File: css/at-rules/keyframes.json [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+              {
+                "version_added": "9"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "4"
+              }
+            ],
+            "safari_ios": {
+              "version_added": null
```
Comment (teoli2003) [inline css/at-rules/keyframes.json]: Set 'true'. Also add a prefix "-webkit-" version, also set to 'true'

File: css/at-rules/keyframes.json [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+              {
+                "prefix": "-moz-",
+                "version_added": "5"
+              }
+            ],
+            "ie": {
+              "version_added": "10"
+            },
+            "ie_mobile": {
+              "version_added": null
```
Comment (teoli2003) [inline css/at-rules/keyframes.json]: Set to 'true'

File: css/at-rules/keyframes.json [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+            },
+            "opera": [
+              {
+                "version_added": "12.10"
+              },
```
Comment (teoli2003) [inline css/at-rules/keyframes.json]: Add an extra entry for the -webkit- prefixing story:
{
  "version_added": "15",
  "prefix": "-webkit-"
},

File: css/at-rules/keyframes.json [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "16"
+              },
+              {
```
Comment (teoli2003) [inline css/at-rules/keyframes.json]: We need to add two {} blocks to describe the story of -webkit- prefixes in Firefox (added in 44 behind a pref, without a pref in 49)

{
  "version_added": "49",
  "prefix": "-webkit-"
},
{
  "version_added": "44",
  "prefix": "-webkit-",
  "flag": {
    "type": "preference",
    "name": "layout.css.prefixes.webkit",
    "value_to_set": "true"
  }
},

File: css/at-rules/keyframes.json [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+                "prefix": "-moz-",
+                "version_added": "5"
+              }
+            ],
+            "firefox_android": [
```
Comment (teoli2003) [inline css/at-rules/keyframes.json]: Same as Firefox (-webkit- prefixes)

File: css/at-rules/keyframes.json [hunk 9]
```diff
@@ -0,0 +1,134 @@
+{
+  "css": {
+    "at-rules": {
+      "@keyframes": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@keyframes",
+          "support": {
+            "webview_android": {
```
Comment (teoli2003) [inline css/at-rules/keyframes.json]: Added unprefixed version in "43" too.

File: css/at-rules/keyframes.json [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+            "chrome": [
+              {
+                "version_added": "43"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "chrome_android": {
```
Comment (teoli2003) [inline css/at-rules/keyframes.json]: Also supported (same as "chrome")

File: css/at-rules/keyframes.json [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": null
+              },
+              "safari_ios": {
+                "version_added": null
+              }
+            }
```
Comment (teoli2003) [inline css/at-rules/keyframes.json]: Add a "status" box. (This is standard behaviour; the spec says 'Animations override all normal rules, but are overriden by !important rules. ')

**Comment by teoli2003** [comment]
@Elchi3 Quick question: is it ok to have a dash in the identifier (and in the directory name):
```
"css": {
    "at-rules": {
      "@keyframes": {
```
?

**Comment by Elchi3** [comment]
Yes, that should be alright. The mdn/data repo also uses "at-rules" and accessing the data using "css.at-rules.@keyframes" should work just fine.

**Comment by ddbeck** [comment]
OK, I think this one is ready for another look, @teoli2003.

This did remind me of a question, too: I've seen tables with Android Webview version numbers that seem to be Android version numbers (e.g., `4.3`, `4.4`, `5`, etc.) and some that correspond to Chrome version numbers (e.g., `36`, `43`, etc.)? What's the right way to migrate Webview? Keep the number regardless of the sequence they belong to? Go to `true` when it's an Android version number?

**Comment by ddbeck** [comment]
Also, I agree this is a grim and scary story, but at least it's the right time of year for it. 🎃

**Comment by teoli2003** [comment]
Before  Android 4.4, the Webview wasn't based on Chromium and has only the Android version. From Android 4.4, it is based on Chromium (like 4.4 is based on Chromium 30). So Webview should not be the Android version for 4.4 or larger.

@Elchi3 is adding version validation and I think the Webview versions will added to these checks at some point. Meanwhile, I would say to not lose information (I prefer a 5 that we will convert to a Chrome-based  version later to a true that will stay like this forever).

**Pull request merged**